docs/content_schemas/changing-an-existing-content-schema.md

```diff
@@ -1,50 +1,64 @@
 # Changing an existing content schema
 
-## General workflow
+## When `default_format.jsonnet` does not need updating
 
-* create a branch
-* make changes to the relevant schema fragment under `formats/<format name>.jsonnet`
-* change the relevant frontend examples (or add a new example)
-* run `rake build_schemas` to compile a new schema.json
-* run `rake` to test the example against the schema
-* commit and push
-* open a PR
-* watch the multi-build status to see if all apps are compatible with the change
-* if the builds are successful, it can be merged
+This describes how to add or remove fields that extend the top level fields
+defined in `default_format.jsonnet`. For example, if you wanted to submit an
+additional field via a form in a publishing app, you might add a field nested
+within `details` (a field already defined in `default_format.jsonnet`).
 
-## Adding a new field to a content schema
+1. If you're adding or removing a required field:
+   - Update publishing app(s) to provide/stop providing the field
+   - If removing a field, update frontend app(s) to no longer access the
+     soon-to-be-removed field
+   - Update existing data in Content Store to ensure they conform with the
+     incoming schema changes
+1. Update the relevant schema fragment under
+   `content_schemas/formats/<format_name>.jsonnet`
+   - Add or remove the field under the `details` > `properties` property
+   - If the field is/was required, update the `details` > `required` array
+1. Find or create the relevant definition file within
+   `content_schemas/formats/shared/definitions` and add or remove the definition
+   for the field
+1. Add, remove, or update the relevant example schemas in
+   `content_schemas/examples/**/<example_name>.jsonnet`.
+1. Run `rake build_schemas` to compile the updated schemas in
+   `content_schemas/dist/` and validate the example schemas.
+1. Check that all the CI workflows pass. These will verify that the content
+   schemas are compatible with the suite of publishing apps
 
-Follow the General workflow described above. You are now free to add support for the field to apps. If you are adding a mandatory field you will need to additionally:
+## When `default_format.jsonnet` needs updating
```

Comment:
Similarly, something like "When adding or removing top-level fields" might read better here

Reply:
We wrote this and then deleted it a few times, useful to have another opinion! I think we can put it back.